src/pool/srv_pool_scrub.c

```diff
@@ -292,22 +290,12 @@ ds_start_scrubbing_ult(struct ds_pool_child *child)
 	/* There will be several levels iteration, such as pool, container, object, and lower,
 	 * and so on. Let's use DSS_DEEP_STACK_SZ to avoid ULT overflow.
 	 */
-	rc = dss_ult_create(scrubbing_ult, child, DSS_XS_SELF, 0, DSS_DEEP_STACK_SZ, &thread);
-	if (rc) {
-		D_ERROR(DF_UUID"[%d]: Failed to create Scrubbing ULT. %d\n",
-			DP_UUID(child->spc_uuid), dmi->dmi_tgt_id, rc);
-		return rc;
-	}
-
-	D_ASSERT(thread != ABT_THREAD_NULL);
-
 	sched_req_attr_init(&attr, SCHED_REQ_SCRUB, &child->spc_uuid);
-	attr.sra_flags = SCHED_REQ_FL_NO_DELAY;
```

Comment:
Was this flag dropped intentionally?

Reply:
Yes, I think scrub ULT shouldn't have this flag. @ryon-jensen , is that correct?